### DISCUSSION TITLE
ribbion 能修改吗

### DISCUSSION THREAD

**Discussion by sunyubk** [liuyib/hexo-theme-stun#196] [Q&A（反馈问题、提问、报告 Bug 等）] [answered]
ribbion想换一个可以动的，但是没找到在哪里修改，请问在哪里改？

**Comment by liuyib** [comment, marked as answer]
文档里搜“特效”：

> 文档地址：https://theme-stun.github.io/docs/zh-CN/

![image](https://user-images.githubusercontent.com/38221479/118600597-cbd31680-b7e3-11eb-9a81-defb1cb8744a.png)
